### ISSUE TITLE
[Feature Request] Button and MenuOption href

### ISSUE THREAD

**Issue by dereksdev** [JosephusPaye/Keen-UI#259]
Buttons and MenuOptions should be able to take an href attribute and use an anchor element.

**Comment by JosephusPaye** [comment]
Added in 0ccb0e0273c89ebff9bac5681884e48827450a9f and 5d52b7ed0973114841dafc2d3b8b72a1035592b5, coming soon.

To try it:

```sh
# npm
npm install keen-ui@JosephusPaye/Keen-UI#next

# yarn
yarn add keen-ui@JosephusPaye/Keen-UI#next
```

See the [changelog](https://github.com/JosephusPaye/Keen-UI/blob/next/CHANGELOG.md) for all changes.

**Comment by JosephusPaye** [comment]
Fix released in [v1.1.0](https://github.com/JosephusPaye/Keen-UI/releases/tag/v1.1.0).
